launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/NewLauncherProjectWizardPage.java

```diff
@@ -0,0 +1,195 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.launcher.reddeer.wizards;
+
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.DefaultCombo;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.swt.widgets.Composite;
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+public class NewLauncherProjectWizardPage extends WizardPage {
+	
+	@Override
+	public void createControl(Composite arg0) {
+		// TODO Auto-generated method stub
+		
+	}
+	
+	public NewLauncherProjectWizardPage(String pageName) {
+		super(pageName);
+		// TODO Auto-generated constructor stub
+	}
+
+	protected final static Logger log = Logger
+			.getLogger(NewLauncherProjectWizardPage.class);
+
+	/**
+	 * Finishes this wizard without possibility to check
+	 * NewProjectExamplesReadyPage.
+	 */
+	public void finish() {	
```

Comment:
@zcervink I would suggest to have 2 finish methods: "public void finish()" and "public void finish(TimePeriod period)". The first one would call the second one with some default value, like:
public void finish() {
         finish(TimePeriod.DEFAULT);
}
Afterwards you can simplify code in tests.

Reply:
Changed. Now testing on Jenkins.